src/pint/utils.py

```diff
@@ -1265,6 +1265,98 @@ def split_swx(model, time):
     return index, newindex
 
 
+def wavex_setup(fitter, freqs=None, n_freqs=None):
```

Comment:
Is doing this on a `fitter` object needed?  Or would it be better to do it to a `model` object but where you have to specify the time span if using `n_freqs`?  I think there are arguments both ways.

Reply:
Changed it to act on a `model` object where the user specifies a `T_span`